backend/analytics_server/mhq/store/repos/code.py

```diff
@@ -367,6 +367,32 @@ def get_team_repos(self, team_id) -> List[OrgRepo]:
         team_repo_ids = [tr.org_repo_id for tr in team_repos]
         return self.get_repos_by_ids(team_repo_ids)
 
+    @rollback_on_exc
+    def get_prs_merged_without_review(
+        self, team_id: str, interval: Interval, pr_filter: PRFilter = None
+    ) -> List[PullRequest]:
+        AllOrg = self.get_team_repos(team_id)
+        AllOrg_ids = [tr.id for tr in AllOrg]
+        query = (
+            self._db.session.query(PullRequest)
+            .filter(PullRequest.repo_id.in_(AllOrg_ids))
+            .filter(PullRequest.state == PullRequestState.MERGED)
+            .filter(PullRequest.merge_time == None)
+            .filter(
+                or_(
+                    PullRequest.created_at.between(
+                        interval.from_time, interval.to_time
+                    ),
+                    PullRequest.updated_at.between(
+                        interval.from_time, interval.to_time
+                    ),
+                )
```

Comment:
since we are fetching PRs merged without review in interval. I think it would be valuable to look into `_filter_prs_merged_in_interval` method

Reply:
@samad-yar-khan will replace it with already implemented method , did see this while scrolling the file